examples/linear_model/plot_ridge_coeffs.py

```diff
@@ -0,0 +1,86 @@
+"""
+==============================================================
+Plot Ridge coefficients as a function of the L2 regularization
+==============================================================
+
+.. currentmodule:: sklearn.linear_model
+
+:class:`Ridge` Regression is the estimator used in this example.
+Each color in the left plot represents a different feature of the
+coefficient vector, and this is displayed as a function of the
+regularization parameter. The right plot shows how exact the solution
```

Comment:
Can you add a line somewhere stating how this is different from the other example?